src/filter/filters.ts

```diff
@@ -138,28 +138,32 @@ export const handleRegex = (context: any, filter: string): boolean => {
  * @returns The decoded ABI.
  */
 export const handleAbiDecode = (context: any, filter: { $abi: Abi }) => {
-  const sighash = slice(context, 0, 4)
+  try {
+    const sighash = slice(context, 0, 4)
 
-  const { functionName, args = [] } = decodeFunctionData({
-    abi: filter.$abi,
-    data: context,
-  })
+    const { functionName, args = [] } = decodeFunctionData({
+      abi: filter.$abi,
+      data: context,
+    })
 
-  const abiItem = getAbiItem({
-    abi: filter.$abi,
-    name: functionName,
-    args,
-  })
+    const abiItem = getAbiItem({
+      abi: filter.$abi,
+      name: functionName,
+      args,
+    })
 
-  const namedArgs = [...abiItem.inputs].reduce(
-    (acc: Record<string, any>, input, index) => {
-      acc[`${input.name || index}`] = args[index]
-      return acc
-    },
-    {},
-  )
+    const namedArgs = [...abiItem.inputs].reduce(
+      (acc: Record<string, any>, input, index) => {
+        acc[`${input.name || index}`] = args[index]
+        return acc
+      },
+      {},
+    )
 
-  return { ...namedArgs, sighash, functionName }
+    return { ...namedArgs, sighash, functionName }
+  } catch (_e) {
+    return null
```

Comment:
Imo I think we should log the error since it might be helpful on any decoding. Wondering if it makes sense to try to only log when error isn't `AbiFunctionSignatureNotFoundError`

So maybe can do something like

```
} catch (e) {
  if (typeof e === 'AbiFunctionSignatureNotFoundError') {
    return null
  }
  
  console.error(e)
}
```

^ not sure if this is correct way to get the specific error type